native/html5ever_nif/Cargo.toml

```diff
@@ -7,10 +7,10 @@ edition = "2018"
 [lib]
 name = "html5ever_nif"
 path = "src/lib.rs"
-crate-type = ["dylib"]
+crate-type = ["cdylib"]
 
 [dependencies]
-rustler = "^0.22"
+rustler = { git = "https://github.com/rusterlium/rustler" }
```

Comment:
We should wait for a new release of `rustler` since the current version (0.22) did not work on Windows (It's probably the OTP that is not installed).